packages/ui/src/helpers/authenticator/__tests__/listenToAuthHub.test.ts

```diff
@@ -18,10 +18,12 @@ const unauthenticatedStateMachine = {
 };
 
 const sendSpy = jest.spyOn(authenticatedStateMachine, 'send');
+const unAuthSendSpy = jest.spyOn(unauthenticatedStateMachine, 'send');
```

Comment:
NIT: let's align the spy names:
```suggestion
const authSendSpy = jest.spyOn(authenticatedStateMachine, 'send');
const unauthSendSpy = jest.spyOn(unauthenticatedStateMachine, 'send');
```

Reply:
Sounds good